kore/src/Kore/Log.hs

```diff
@@ -292,10 +297,16 @@ swappableLogger mvar =
     release = liftIO . putMVar mvar
     worker a logAction = Colog.unLogAction logAction a
 
-defaultLogPretty :: SomeEntry -> Pretty.Doc ann
-defaultLogPretty (SomeEntry entry) =
+defaultLogPretty :: ActualEntry -> Pretty.Doc ann
+defaultLogPretty ActualEntry { actualEntry = SomeEntry entry, entryContext } =
     header Pretty.<+> longDoc entry
   where
     severity = prettySeverity (entrySeverity entry)
     type' = Pretty.pretty (lookupTextFromTypeWithError $ toSomeEntryType entry)
-    header = severity Pretty.<+> Pretty.parens type' <> Pretty.colon
+    header =
+        Pretty.brackets (Pretty.hsep prettyContext)
```

Comment:
We should use `Pretty.vsep` here and put the context before the brackets. My intention was to have something that looks like a "traditional" compiler error message:

```
while applying rule ...:
in side condition ...:
while doing something else ...:
Error: ...
    ...
```

Reply:
I changed this to look something like the following for entries with `shortDoc`: 
```
kore-exec: [2020-03-12 12:29:53.188880665]
  While starting axiom evaluation of Lbl'Unds-GT-Eqls'Int'Unds' :
  While starting axiom evaluation of Lbl'Unds'orBool'UndsUnds'BOOL'Unds'Bool'Unds'Bool'Unds'Bool :
    Debug (DebugAxiomEvaluation): Ending with N/A: Lbl'Unds-GT-Eqls'Int'Unds'
```
Ordinary entries still look the same:
```
kore-exec: [2020-03-12 12:29:53.19308644] Debug (DebugAxiomEvaluation): Starting:
Lbl'Unds'orBool'UndsUnds'BOOL'Unds'Bool'Unds'Bool'Unds'Bool
```
Is this what you had in mind?